blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -383,11 +384,14 @@ export const actions = {
                         // run. No need to create another i.e. ignore this event.
                         return;
                     }
+                    if (parseInt(run.id, 10) > nextId) { // need the estimated next id
+                        nextId = parseInt(run.id, 10);
+                    }
                 }
```

Comment:
The queueId has always been on all events and was they way in which we correlated between queued and running runs (the queueId would always be the same).

It really depends on why the above change was made. There may be a reason but would don't know without a comment.